src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -67,6 +68,7 @@ public function singularifyProvider()
             array('emphases', array('emphas', 'emphase', 'emphasis')),
             array('faxes', 'fax'),
             array('feet', 'foot'),
+            array('feedback', 'feedback'),
```

Comment:
@WouterJ I think the question was more related to problems in how property accessor uses singular/plural forms, and possible issues when these forms are the same.